core/src/main/java/org/jdbi/v3/core/statement/SqlStatements.java

```diff
@@ -143,6 +145,7 @@ public TimingCollector getTimingCollector() {
      * @param timingCollector the new timing collector
      * @return this
      */
+    @Deprecated
```

Comment:
Can we get a note added to javadoc as well? e.g. `@deprecated use {@link #setSqlLogger(SqlLogger)} instead` or something like that?